packages/web/components/pool-detail/base-pool.tsx

```diff
@@ -87,7 +87,7 @@ export const BasePoolDetails: FunctionComponent<{
                     {t("pool.swapFee")}
                   </span>
                   <h4 className="text-osmoverse-100">
-                    {new RatePretty(pool.swapFee).maxDecimals(0).toString()}
+                    {new RatePretty(pool.swapFee).maxDecimals(2).toString()}
```

Comment:
Why did this change?

Reply:
Ah this would never work, since its always 1 to 2 decimal points.

Reply:
Is there somewhere that makes this a mobile only component, or did this not work on desktop either?

Reply:
We have `isMobile` that we could use